css/style.css

```diff
@@ -448,7 +480,39 @@ body {
   background-color: #008552;
 }
 
+.footer {
+  border-top: 1px solid #aeb3bb;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+}
+
 @media only screen and (max-width: 600px) {
+  .toolbar {
+    background: white;
```

Comment:
- [ ] Kindly make sure to use a proper media query for the desktop which has to be between `768px` and `1440px`, as per [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=(Desktop%3A%20768px%20or%20wider).). 
- [ ] please use `min-width` instead of `max-width`, as this project is built mobile first.